### DISCUSSION TITLE
Vuegraf.json - Disabling Per Second Details

### DISCUSSION THREAD

**Discussion by toshioue** [jertel/vuegraf#137] [Q&A] [answered]
I have a quesation on the documentation regarding `detailedDataEnabled: true` in the vuegraf.json. 

If this setting is set to "false" - will this still pull hourly data points into the DB and enable the filter:`detailed = Hour represents the data summarized in hours` ?

I'm hoping to restrict data to only hourly if possible to minimize DB storage needed and wondering if it's possible with the setting `detailedDataEnabled:` to `false`?

**Comment by jertel** [comment, marked as answer]
No, `detailedDataEnabled: false` will only stop the per-second data from being ingested once per hour. The standard per-minute data will continue to be stored. Hourly data is only recorded during history data collection, which is usually a one time operation when you first start using Vuegraf.

That said, storing per-minute data is really not going to use a lot of data. It will be in the range of a few megabytes per month for a 16 sensor unit.
